velero-plugins/imagestream/backup.go

```diff
@@ -262,7 +246,7 @@ func internalRegistrySystemContext() (*types.SystemContext, error) {
 
 	ctx := &types.SystemContext{
 		DockerDaemonInsecureSkipTLSVerify: true,
-		DockerInsecureSkipTLSVerify: types.OptionalBoolTrue,
+		DockerInsecureSkipTLSVerify:       types.OptionalBoolTrue,
```

Comment:
Any idea why this whitespace was added? It might just be your text editor that added this... Go formatters are all different :)

Reply:
That's my text editor, it wants to align attribute values. I can disable it if needed.

Reply:
Oh duh, It's lining up with `true`. Sorry I missed that. LGTM!